utils/inflectors.ts

```diff
@@ -1,4 +1,14 @@
 export function titleize(str: string) {
+  return str
+    .replace(/^[\s_]+|[\s_]+$/g, "")
+    .replace(/[_\s]+/g, " ")
+    .replace(/-/g, " ")
+    .replace(/^[\w]|[ _-]\w/g, (m) => {
+      return m.toUpperCase();
+    });
+}
+
+export function breadcrumbTitleize(str: string) {
```

Comment:
We could probably make titleize call breadcrumbTitleize, and include the `.replace(/-/g, " ")` portion.
More DRY.

Reply:
removed this function... the only places using it were also calling .replace() to remove hyphens immediately